src/mono/wasm/debugger/DebuggerTestSuite/DateTimeTests.cs

```diff
@@ -42,21 +38,13 @@ public async Task CheckDateTimeLocale(string locale)
                        DateTimeFormatInfo dtfi = CultureInfo.GetCultureInfo(locale).DateTimeFormat;
                        CultureInfo.CurrentCulture = new CultureInfo(locale, false);
                        DateTime dt = new DateTime(2020, 1, 2, 3, 4, 5);
-                       string dt_str = dt.ToString();
-
-                       var fdtp = dtfi.FullDateTimePattern;
-                       var ldp = dtfi.LongDatePattern;
-                       var ltp = dtfi.LongTimePattern;
-                       var sdp = dtfi.ShortDatePattern;
-                       var stp = dtfi.ShortTimePattern;
 
                        CheckString(locals, "fdtp", fdtp);
                        CheckString(locals, "ldp", ldp);
                        CheckString(locals, "ltp", ltp);
                        CheckString(locals, "sdp", sdp);
                        CheckString(locals, "stp", stp);
                        await CheckDateTime(locals, "dt", dt);
```

Comment:
We can now change this to:

```
await CheckProps(locals, new
{
   fdtp = TString(fdtp),
   ldp = TString(ldp),
   ltp = TString(ltp),
   sdp = TString(sdp),
   stp = TString(stp),
   dt = TDateTime(new DateTime(2020, 1, 2, 3, 4, 5))
}, "locals", num_fields: 8);
```
(Just typed that code here, so make sure to run the test 😬 )
And you can remove `dt_str` from `debugger-datetime-test.cs` also, since we are not using that now.

Reply:
Done!